[Bug]: Duplicate default_otp_enabled function in config/schema.rs causes compile failure

### Summary

 cargo build fails with error[E0428]: the name 'default_otp_enabled' is defined multiple times after pulling latest   dev branch.

### Affected component

runtime/daemon

### Severity

S1 - workflow blocked

### Current behavior


  error[E0428]: the name `default_otp_enabled` is defined multiple times
      --> src/config/schema.rs:4561:1
       |
  4557 | fn default_otp_enabled() -> bool {
       | -------------------------------- previous definition of the value `default_otp_enabled` here
  ...
  4561 | fn default_otp_enabled() -> bool {
       | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ `default_otp_enabled` redefined here

### Expected behavior

default_otp_enabled() should be defined exactly once. cargo build should succeed without redefinition errors.

### Steps to reproduce

```bash
cargo build
```

### Impact

  - Affected users: all contributors pulling latest
  - Frequency: always (deterministic compile error)
  - Consequence: completely blocks build, install, and all downstream development

### Logs / stack traces

```text

```

### ZeroClaw version

v0.1.7

### Rust version

1.84

### Operating system

ubuntu 20

### Regression?

Unknown

### Pre-flight checks

- [x] I reproduced this on the latest main branch or latest release.
- [x] I redacted secrets/tokens from logs.
- [x] I removed personal identifiers and replaced identity-specific data with neutral placeholders.

--- a/src/config/schema.rs
+++ b/src/config/schema.rs
@@ -4558,10 +4558,6 @@ fn default_otp_enabled() -> bool {
     true
 }
 
-fn default_otp_enabled() -> bool {
-    true
-}
-
 fn default_otp_token_ttl_secs() -> u64 {
     30
 }

